CVLByExample/Import/certora/specs/base.spec

```diff
@@ -1,52 +1,48 @@
 methods {
-	function minusSevenSomeUInt() external envfree;
-	function someUInt() external returns (uint256) envfree;
+    function minusSevenSomeUInt() external envfree;
+    function someUInt() external returns (uint256) envfree;
 }
 
 /// @notice Invariant in Base to be overridden by importing in `Sub` contract.
 invariant invInBase() someUInt() >= 7 {
-	preserved {
-		require (someUInt() < 30); // @notice Explicit preserved block
-	}
-}
+    // @notice Explicit preserved block
+    preserved {
+        require (someUInt() < 30);
```

Comment:
MUST: Same as for `assert` (and for `satisfy`!), a `require` should allow to be followed by a comment. (Even though we now support `require(<condition>, <reason>)` I believe, folks will keep writing comments afterwards - and we want it to maintain existing specs still correctly formatted.